### PR TITLE
[tests] Delete disk which already deleted

### PR DESCRIPTION
**Pull request by pkulax**

### CODE REVIEW

File: build/storage/tests/it/ptf_tests/tests/test_hot_plug.py [hunk 1]
```diff
@@ -63,6 +63,23 @@ def runTest(self):
...
+            self.host_target_platform.get_service_address(),
+            remote_nvme_storages,
+            range(len(remote_nvme_storages)),
+        )
+        fake_delete_responses = self.ipu_storage_platform.delete_virtio_blk_devices(
+            fake_devices_handles
+        )
+        for response in fake_delete_responses:
+            self.assertEqual(response, "False")
+
```
Comment (artek-koltun) [inline build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:82]: Commit header is too long. Would it make sense to separate it to 2?

File: build/storage/tests/it/ptf_tests/system_tools/test_platform.py [hunk 2]
```diff
@@ -255,6 +255,24 @@ def create_virtio_blk_devices(
             )
         return device_handles
 
+    def create_fake_virtio_blk_devices(
```
Comment (artek-koltun) [inline build/storage/tests/it/ptf_tests/system_tools/test_platform.py:258]: It is not fake virtio-blk device. Here you are creating virtio-blk devices with invalid device handles, aren't you?

File: build/storage/tests/it/ptf_tests/tests/test_hot_plug.py [hunk 3]
```diff
@@ -63,6 +63,23 @@ def runTest(self):
...
+        fake_devices_handles = self.ipu_storage_platform.create_fake_virtio_blk_devices(
+            self.host_target_platform.get_service_address(),
+            remote_nvme_storages,
+            range(len(remote_nvme_storages)),
+        )
+        fake_delete_responses = self.ipu_storage_platform.delete_virtio_blk_devices(
+            fake_devices_handles
+        )
+        for response in fake_delete_responses:
+            self.assertEqual(response, "False")
```
Comment (artek-koltun) [inline build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:81]: Why do we use compare here strings? It sounds like error prone operation. I believe we have to return False as boolean type from `delete`

Comment (pkulax) [reply build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:81]: paramiko terminal return string - should I map this string to bool?

File: build/storage/tests/it/ptf_tests/system_tools/test_platform.py (hunk 2 above)
Comment (pkulax) [reply build/storage/tests/it/ptf_tests/system_tools/test_platform.py:258]: I want to not create virtio blk, I want delete non exist virtio_blk_device but now we have dedicated class so I create class but this class not create virtio_blk -

File: build/storage/tests/it/ptf_tests/tests/test_hot_plug.py (hunk 1 above)
Comment (pkulax) [reply build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:82]: when I will correct all issues I split this pr

File: build/storage/tests/it/ptf_tests/tests/test_hot_plug.py (hunk 3 above)
Comment (artek-koltun) [reply build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:81]: Yes, please

File: build/storage/tests/it/ptf_tests/system_tools/test_platform.py (hunk 2 above)
Comment (artek-koltun) [reply build/storage/tests/it/ptf_tests/system_tools/test_platform.py:258]: I understand. They are not fake. Fake word is [associated ](https://testing.googleblog.com/2013/07/testing-on-toilet-know-your-test-doubles.html)with test doubles and it provides a lightweight implementation of an API that behaves like the real implementation, but isn't suitable for production.
Here you need just test what happens with invalid device handle. No device creation is performed.
So, would it be even easier to write your test like:
```
virtio_blk_with_invalid_device_handle = VirtioBlkDevice(
        f"some_invalid_device_handle_{volume}_{physical_id}",
        volume,
        self.ipu_storage_platform,
        host_target_address_service,
    )
delete_result = self.ipu_storage_platform.delete_virtio_blk_device(virtio_blk_with_invalid_device_handle)
self.assertFalse(delete_result )
```

File: build/storage/tests/it/ptf_tests/tests/test_hot_plug.py (hunk 1 above)
Comment (pkulax) [reply build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:82]: done

File: build/storage/tests/it/ptf_tests/system_tools/test_platform.py (hunk 2 above)
Comment (pkulax) [reply build/storage/tests/it/ptf_tests/system_tools/test_platform.py:258]: deleted

File: build/storage/tests/it/ptf_tests/tests/test_hot_plug.py (hunk 3 above)
Comment (pkulax) [reply build/storage/tests/it/ptf_tests/tests/test_hot_plug.py:81]: changed

File: build/storage/tests/it/ptf_tests/system_tools/test_platform.py [hunk 4]
```diff
@@ -267,8 +267,10 @@ def create_virtio_blk_devices_sequentially(
         )
 
     def delete_virtio_blk_devices(self, devices_handles):
-        for device_handle in devices_handles:
-            device_handle.delete(self.cmd_sender)
+        responses = [
+            device_handle.delete(self.cmd_sender) for device_handle in devices_handles
```
Comment (artek-koltun) [inline build/storage/tests/it/ptf_tests/system_tools/test_platform.py:271]: Is it possible to return delete result as boolean not string value?

Comment (pkulax) [reply build/storage/tests/it/ptf_tests/system_tools/test_platform.py:274]: paramiko return string but i changed it in delete function

**Review by artek-koltun — APPROVED** [review]

**Pull request merged**